packages/mds-agency/types.ts

```diff
@@ -11,8 +11,7 @@ export type AgencyApiRequest<P extends Params = ParamsDictionary> = ApiRequest<P
 
 export type AgencyApiAccessTokenScopes = 'admin:all' | 'vehicles:read'
 
-export type AgencyApiResponse<TBody = any> = ApiVersionedResponse<
-  AGENCY_API_SUPPORTED_VERSION,
+export type AgencyApiResponse<TBody = any> = ApiResponse<
```

Comment:
I think you can remove the `= any` (and thus the eslint warning) since a type is always specified below. Also, `unknown` is preferable to `any` for _don't care_ scenarios.